src/mon/OSDMonitor.cc

```diff
@@ -4384,6 +4384,32 @@ void OSDMonitor::get_pools_health(
 	detail->push_back(make_pair(HEALTH_WARN, ss.str()));
     }
 
+    if (pool.size <= g_conf->mon_pool_warn_size) {
+      stringstream ss;
+      ss << "pool '" << pool_name << "' size below the config warn size";
+      summary.push_back(make_pair(HEALTH_WARN, ss.str()));
+      if (detail) {
+        stringstream ss;
+        ss << "pool '" << pool_name << "' size is '" << pool.size 
+           << "' below the config warn size ( " << g_conf->mon_pool_warn_size << " )"
```

Comment:
..and similar with the other one.  that way they see what the config value is to adjust.